alkos/kernel/arch/x86_64/abi/terminal/terminal.cpp

```diff
@@ -1,48 +1,71 @@
 /* internal includes */
-#include <terminal.hpp>
 #include <defines.hpp>
+#include <terminal.hpp>
 
 #ifdef __DEBUG_TERMINAL_TEST__
 #include <serial_port_qemu/serial_qemu.hpp>
-#endif // __DEBUG_TERMINAL_TEST__
+#endif  // __DEBUG_TERMINAL_TEST__
```

Comment:
I only realized this now, but maybe rename this to just DEBUG_TERMINAL
because I genuinly thought it's a flag used when testing the debug terminal